### DISCUSSION TITLE
Health adjacent data

### DISCUSSION THREAD

**Discussion by daniellebitt** [TheLightCollective/PatientLedAI#9] [Ideas]
It would be great if future iterations extend to health-adjacent data from social media, wearables, phone logs, recordings, etc that may be used by companies and researchers and which can be extremely sensitive and identifying (even if the data are "de-identified").

**Comment by mauriciofuhrman** [comment]
Hi Daniel, I have an API to do just this. Let me know if you are interested in pulling all your health data from apple health, fitbit, whoop, and oura ring.
